cypress-tests/cypress/e2e/PaymentUtils/Stripe.js

```diff
@@ -501,15 +515,15 @@ export const connectorDetails = {
         },
       },
     },
-    MITAutoCapture: {
-      Request: {},
-      Response: {
-        status: 200,
-        body: {
-          status: "succeeded",
+    MITAutoCapture: getCustomExchange({
+      Configs: {
+        CONNECTOR_CREDENTIAL: {
+          specName: ["connectorAgnostic"],
```

Comment:
what is the use of this specName: ["connectorAgnostic"] array? 
when do you use this specName config, is there a way to make it intuitive and self explanatory?

Reply:
glad you asked!

we pass `specName` within `Configs` when we want to restrict a connector credential to a specific request / exchange.

in this case, `connectorAgnostic` file (`00024`) will have `connector_2` passed in every `MITAutoCapture` call that is made. and in mandates, if used, `connector_2` will not be passed and hence the default will be sent (`connector_1`).